### PR TITLE
dtos and mappers inventory update

### PR DESCRIPTION
**Pull request by Camilocastellanos1002**

### CODE REVIEW

File: src/main/java/com/riwi/localstorage/riwi_local_storage/infrastructure/mappers/InventaryDisableMapper.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+import com.riwi.localstorage.riwi_local_storage.api.dto.request.create.InventaryRequest;
+import com.riwi.localstorage.riwi_local_storage.api.dto.response.InventaryResponse;
+import com.riwi.localstorage.riwi_local_storage.domain.entities.Inventory;
+
+@Mapper(componentModel = MappingConstants.ComponentModel.SPRING)
+public interface InventaryDisableMapper {
+    
+    @Mappings({
+        @Mapping(target = "id",ignore = true),
```
Comment (juansefdz) [inline src/main/java/com/riwi/localstorage/riwi_local_storage/infrastructure/mappers/InventaryDisableMapper.java]: @Mapping(target = "id",ignore = true) delete this

File: src/main/java/com/riwi/localstorage/riwi_local_storage/infrastructure/mappers/InventaryCreateMapper.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import org.mapstruct.MappingConstants;
+import org.mapstruct.Mappings;
+
+import com.riwi.localstorage.riwi_local_storage.api.dto.request.create.InventaryRequest;
+import com.riwi.localstorage.riwi_local_storage.api.dto.response.InventaryResponse;
+import com.riwi.localstorage.riwi_local_storage.domain.entities.Inventory;
+
+@Mapper(componentModel = MappingConstants.ComponentModel.SPRING)
+public interface InventaryCreateMapper {
+    @Mappings({
```
Comment (juansefdz) [inline src/main/java/com/riwi/localstorage/riwi_local_storage/infrastructure/mappers/InventaryCreateMapper.java]: delete this 
    @Mappings({
        @Mapping(target = "id",ignore = true),
        @Mapping(target = "saleDetails",ignore = true),
        @Mapping(target = "supplierOrders",ignore = true),
        @Mapping(target = "product",ignore = true),
        @Mapping(target = "branch",ignore = true)

    })

**Review by juansefdz — CHANGES_REQUESTED** [review]
please verify the requested changes

File: src/main/java/com/riwi/localstorage/riwi_local_storage/api/dto/request/create/InventaryRequest.java
Comment (SMoreno73) [inline src/main/java/com/riwi/localstorage/riwi_local_storage/api/dto/request/create/InventaryRequest.java:1]: Don´t use @Data if you put setter and getter notation that's enough

**Review by SMoreno73 — CHANGES_REQUESTED** [review]

**Review by juansefdz — APPROVED** [review]

**Pull request merged**